### PR TITLE
decision-services-quickstarts module

### PR DESCRIPTION
**Pull request by jomarko**
The added kogito-decision-services-quickstarts module should be used for testing submodules:
- kogito-examples
- optaplanner-quickstarts
    
The added kogito-decision-services-quickstarts-optawebs module should be used for testing submodules:
- optaweb-vehidle-routing
- optaweb-employee-rostering

### CODE REVIEW

**Comment by jomarko** [comment]
https://rhba-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/custom/job/jstastny/job/run-tests-generated/38

File: run-tests-with-build-kogito-decision-services-quickstarts-optawebs/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Execution
+-------------------------------------
+`mvn verify -Doptaweb=$OPTAWEB_NAME -s $PROJECT_SETTINGS -Dkogito.decision.services.quickstarts.url=$QUICKSTARTS_URL`
+
+where 
+
+`$OPTAWEB_NAME` is a profile name {vehicle-routing|employee-rostering} 
+
+`$PROJECT_SETTINGS` is the settings.xml with the repository which contains ${OPTAPLANNER_BUILD_VERSION} tar.gz
```
Comment (jstastny-cz) [inline run-tests-with-build-kogito-decision-services-quickstarts-optawebs/README.md:17]: I think this applies to the original optaplanner-optawebs plugin only. Here we provide exact url to the tar.gz, but the settings are still needed for the dependency download and mainly proxy settings.
I would rephrase maybe similarly to:
`is the settings.xml with the repository which contains project dependencies.`

File: run-tests-with-build-kogito-decision-services-quickstarts-optawebs/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+`$PROJECT_SETTINGS` is the settings.xml with the repository which contains ${OPTAPLANNER_BUILD_VERSION} tar.gz 
+
+`$QUICKSTARTS_URL` redhat distribution decision-services-quickstarts.zip url
+
+`-Dintegration-tests={true|false}` run integration cypress tests on docker
+
+`-Dcontainer.runtime={docker|podman}` change runtime
+
+To set up $PROJECT_SETTINGS file, provide there the mirror repository and npm repository e.g:
```
Comment (jstastny-cz) [inline run-tests-with-build-kogito-decision-services-quickstarts-optawebs/README.md:25]: Generalize to
`In your $PROJECT_SETTINGS file, you might want to provide the mirror repository and/or proxied npm repository for your environment.`

**Review by jstastny-cz — CHANGES_REQUESTED** [review]
Just few notes in readme.

File: run-tests-with-build-kogito-decision-services-quickstarts-optawebs/README.md (hunk 1 above)
Comment (jomarko) [reply run-tests-with-build-kogito-decision-services-quickstarts-optawebs/README.md:17]: yes, agree, will do it.

File: run-tests-with-build-kogito-decision-services-quickstarts-optawebs/README.md (hunk 2 above)
Comment (jomarko) [reply run-tests-with-build-kogito-decision-services-quickstarts-optawebs/README.md:25]: yes, agree with that.

**Review by jstastny-cz — APPROVED** [review]
LGTM, thanks.

**Comment by jstastny-cz** [comment]
@radtriste wdyt about the approach? If no complaints I would go ahead with merge so that we can see the changes in action after weekend :-) .

**Review by radtriste — APPROVED** [review]
working well in the pipeline :)

**Comment by radtriste** [comment]
> working well in the pipeline :)

*test pipeline. and did not break the community tests :D
